homeassistant/components/contec_controllers/strings.json

```
@@ -0,0 +1,21 @@
+{
+  "config": {
+    "step": {
+      "user": {
+        "data": {
+          "controllersIp": "[%key:common::config_flow::data::ip%]",
+          "numberOfControllers": "numberOfControllers",
```

```suggestion
          "numberOfControllers": "Number of controllers",
```

Use `python3 -m script.translations develop` to generate `en.json`